src/components/ChapterHelpText.svelte

```diff
@@ -1,8 +1,77 @@
+  <p class="chapter-help-text">
+    {#if chapterId === "about"}
+      Fill out general information about your product or service. Please provide
+      as much detail as possible. By describing your product, the webform will
+      update to show only the sections you need to fill out. If you need to
+      change this later, change your response product information questions.
+    {:else if chapterId === "success_criteria_level_a"}
+      Provide information about Level A issues with your website or web
+      application. Currently OpenACR only supports WCAG 2.0. The default is
+      web-based, and there are optional fields for <a
+        href="#1.1.1-electronic-docs-editor"
+        on:click={() => openFirstAccordion()}>Documentation</a
+      >,
+      <a
+        href="#1.1.1-authoring-tool-editor"
+        on:click={() => openFirstAccordion()}>Authoring</a
+      >
+      and
+      <a href="#1.1.1-software-editor" on:click={() => openFirstAccordion()}
+        >Software</a
+      >. Fill out as appropriate.
+    {:else if chapterId === "success_criteria_level_aa"}
+      Provide information about how your website or web application fails to
+      meet Level AA requirements. Currently OpenACR only supports WCAG 2.0. The
+      default is web-based, and there are optional fields for <a
+        href="#1.2.4-electronic-docs-editor"
+        on:click={() => openFirstAccordion()}>Documentation</a
+      >,
+      <a
+        href="#1.2.4-authoring-tool-editor"
+        on:click={() => openFirstAccordion()}>Authoring</a
+      >
+      and
+      <a href="#1.2.4-software-editor" on:click={() => openFirstAccordion()}
+        >Software</a
+      >. Fill out as appropriate.
+    {:else if chapterId === "success_criteria_level_aaa"}
+      This section is optional. Consider providing information about Level AAA
+      support in your website or web application. WCAG Level AAA reflect
+      accessibility features that exceed the minimum requirements of Section
+      508. It is useful to describe how your product or service exceeds Level AA
+      requirements. Additionally, some end-users and some projects may need
+      specific AAA requirements to be meet.
+    {:else if chapterId === "functional_performance_criteria"}
+      Describe barriers here if they are not covered in other sections.
+      Highlight the impact on different types of disabilities. Optionally, this
+      section may be used to summarize accessibility features described in more
+      detail elsewhere.
+    {:else if chapterId === "hardware"}
+      If the Information and Communications Technology (ICT) includes hardware,
+      please fill out this section.
+    {:else if chapterId === "software"}
+      If your software product is a web application or is rendered through a web
+      browser, do not fill out this section. Fill out <a
+        href="success_criteria_level_a"
+        on:click={() => (window.onbeforeunload = null)}>Tables 1, 2, and 3</a
```

Comment:
Nice solution here. I was going to suggest the `<Link>` component but this is better example to follow.

Reply:
@qjac minor nitpick here. When I test it, it resets the warning. That is if I change a form input (check a box or type something) then I click that link then click the GSA logo/external link it doesn't give the warning. What do you think?

Reply:
Primary importance is to not lose data... I'll do some more exploring/testing.

Reply:
Full disclosure, I do not know why <Link> would not work. Nor do I really understand why use:link works. But the functionality does seem to fix, the issue with allowing subsequent external links to load without the warning.

Reply:
Looks like `use:link` is a fancy Svelte navigator feature https://github.com/mefechoel/svelte-navigator#link-1, beats me as well but hey it works! Lets run with it.